astropy/tests/command.py

```diff
@@ -10,10 +10,10 @@
 import subprocess
 import sys
 import tempfile
-from distutils import log
 from contextlib import contextmanager
 
 from setuptools import Command
+from distutils import log
```

Comment:
Since it is basically not used except to give some `info` messages, I think it can be either replaced with `import logging as log` or the commands replaced with `print`. In favour of the earlier option is:
https://github.com/python/cpython/blob/164b04c47e61bd35d55e61bc74f9fd646eba81bb/Lib/distutils/log.py#L3-L4

Reply:
I'll leave this as future exercise but I'll look into intersphinx. Thanks for the review!

Reply:
See #10580